modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryFolderInfoItemCapabilitiesProvider.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.info.item.provider;
+
+import com.liferay.info.item.capability.InfoItemCapability;
+import com.liferay.info.item.provider.InfoItemCapabilitiesProvider;
+import com.liferay.layout.page.template.info.item.capability.DisplayPageInfoItemCapability;
+import com.liferay.object.model.ObjectEntryFolder;
+import com.liferay.portal.kernel.util.ListUtil;
+
+import java.util.List;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Marco Leo
+ */
+@Component(
+	property = "item.class.name=com.liferay.object.model.ObjectEntryFolder",
+	service = InfoItemCapabilitiesProvider.class
+)
+public class ObjectEntryFolderInfoItemCapabilitiesProvider
```

Comment:
@adolfopa, @marco-leo, just as with asset libraries, do we want to show the display page for all sites?

Reply:
I've put this under a FF in the other PR, so that at least we can delay the decission.